src/main/java/io/strimzi/kafka/bridge/http/HttpBridge.java

```diff
@@ -294,7 +294,7 @@ private void processProducer(RoutingContext routingContext) {
         SourceBridgeEndpoint source = this.httpBridgeContext.getHttpSourceEndpoints().get(httpServerRequest.connection());
 
         if (source == null) {
-            source = new HttpSourceBridgeEndpoint<>(this.vertx, this.httpBridgeConfig, this.httpBridgeContext,
+            source = new HttpSourceBridgeEndpoint<>(this.vertx, this.httpBridgeConfig,
```

Comment:
I left the field for simmetry with the sink endpoint, maybe used in the future but if findbugs is complaining...

Reply:
We need to create some fake use or remove it to silence it ;-).